File: src/main/java/bot/Bogo.java
package bot;


import model.*;
import model.Player;

public class Bogo extends Player {

    public Bogo(Identifier side, Board board) {
        super(side, board);
    }

    @Override
    public int getColumnOfNextMove() {
        return (int)(Math.random() * board.WIDTH);
    }

    @Override
    public void goodbye(Identifier winner) {
        System.out.println("Ciao");
    }

    @Override
    public String getName() {
        return "Bogo";
    }
}

File: src/main/java/bot/Bogo.kt
package bot

import model.Board
import model.Player
import model.Token

class Bogo(side: Token, board: Board) : Player(side, board) {
    override fun getColumnOfNextMove(): Int {
        return (Math.random() * board.WIDTH).toInt()
    }

    override val name: String = "Bogo"

    override fun onMovePlayed(x: Int) = Unit
}